libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.spec.ts

```diff
@@ -337,6 +375,14 @@ describe('ModalWrapperComponent', () => {
       spectator.dispatchFakeEvent(window, 'ionKeyboardDidHide');
       expect(spectator.component['keyboardVisible']).toBeFalse();
     });
+
+    it('should keep same height, when keyboard is opened', () => {
+      const heightHeyboardClosed = spectator.element.getBoundingClientRect().height;
+      spectator.dispatchFakeEvent(window, 'ionKeyboardDidShow');
```

Comment:
You're not dispatching keyboardHeight in event details, so does this actually test anything? 🤔